app/serializers/group_user_assigned_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class GroupUserAssignedSerializer < BasicUserSerializer
+  include UserPrimaryGroupMixin
```

Comment:
Do we still need the primary group mixin?

Reply:
removed in the latest commit😅